src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -157,6 +160,28 @@ class EltwiseShapeInferFactory : public ShapeInferFactory {
     }
 };
 
+void set_intersection(
+    const std::set<std::vector<element::Type>>& precisions1,
```

Comment:
Please add indent to distinguish fn arguments from fn body.

Reply:
done, thanks